src/view/treeNodes/pullRequestNode.ts

```diff
@@ -126,6 +126,13 @@ export class PRNode extends TreeNode implements vscode.CommentingRangeProvider {
 	protected registerSinceReviewChange() {
 		this._disposables.push(
 			this.pullRequestModel.onDidChangeChangesSinceReview(_ => {
+				if (!this.pullRequestModel.isActive) {
+					this.refresh();
+				}
+			})
+		);
+		this._disposables.push(
+			this.pullRequestModel.onDidChangeChangesSinceReviewActive(async _ => {
```

Comment:
Will refershing on this event work? 

https://github.com/microsoft/vscode-pull-request-github/blob/58e7c742befe7519abc21c082f5475a985bdb66f/src/view/reviewModel.ts#L12-L13

Reply:
To get the review model into this class, you could pass it in here

https://github.com/microsoft/vscode-pull-request-github/blob/58e7c742befe7519abc21c082f5475a985bdb66f/src/view/reviewsManager.ts#L55-L56

Reply:
If this doesn't end up working, we can add another event to the review model: `onDidChangeObsoleteFileChanges`, which should do the trick.